resources/src/asset.rs

```diff
@@ -0,0 +1,99 @@
+use utils;
+/// Permissions around who can own an asset and whether or
+/// not the asset issuer can freeze the asset.
+#[derive(Serialize, Deserialize, Debug)]
+struct Flag {
+    auth_required: bool,
+    auth_revocable: bool,
+}
+
+/// Assets are the units that are traded on the Stellar Network.
+/// An asset consists of an type, code, and issuer.
+/// Any asset can be traded for any other asset.
+/// https://www.stellar.org/developers/horizon/reference/resources/asset.html
+#[derive(Serialize, Deserialize, Debug)]
+pub struct Asset {
+    asset_type: String,
+    asset_code: String,
+    asset_issuer: String,
+    #[serde(deserialize_with = "utils::deserialize_from_str")] amount: f64,
```

Comment:
ugh, i just noticed that rustfmt does this. I think i prefer them on the previous lines, we may want to check in a rustfmt yml for config at some point...  #6 (just created)